src/scripts/main.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+// First promise
+const firstPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject(new Error('First promise was rejected in 3 seconds if not clicked'));
+  }, 3000);
+
+  document.addEventListener('click', (evnt) => {
+    if (evnt.button === 0) {
+      clearTimeout(timeoutId);
+      resolve('First promise was resolved on a left click in the document');
+    }
+  });
+});
+
+// Second promise
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (evnt) => {
+    if (evnt.button === 0 || evnt.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
+
+// Third promise
+const thirdPromise = new Promise((resolve) => {
+  let leftClicked = false;
+  let rightClicked = false;
+
+  document.addEventListener('click', (evnt) => {
```

Comment:
click event not works for right button
```suggestion
  document.addEventListener('mousedown', (evnt) => {
```